spec/problem3_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'problem3'
+describe 'problem3' do
+
+  it 'returns true on arrays with duplicates' do
+    expect(problem3([1,2,2])).to eq true
+    expect(problem3([5,4,4,2,1])).to eq true
+  end
+
+  it 'returns false on arrays with no duplicates' do
+    expect(problem3([])).to eq false
+    expect(problem3([1,2,3])).to eq false
+    expect(problem3([5,4,3,2,1])).to eq false
+  end
```

Comment:
What if you add [4,4,1,1] as a test case ?

Reply:
Then I have to rewrite my solution.